Cargo.toml

```diff
@@ -1,5 +1,10 @@
 [workspace]
-members = ["sbtc-core", "stacks-core"]
+members = [
+    "sbtc-core",
+    "stacks-core",
+    "xtask",
+]
+resolver = "2"
```

Comment:
It's possible this field is unnecessary according to this page:
https://doc.rust-lang.org/edition-guide/rust-2021/default-cargo-resolver.html